state/src/impls/top_level.rs

```diff
@@ -356,7 +356,7 @@ impl TopState for TopLevelState {
         self.top_cache.remove_action_data(key)
     }
 
-    fn update_chain_params(&mut self, chain_params: ChainParams) -> StateResult<()> {
+    fn update_consensus_params(&mut self, chain_params: ChainParams) -> StateResult<()> {
```

Comment:
It seems you reverted https://github.com/CodeChain-io/foundry/pull/535/commits/7e7013bffa488c4c4f64c6612404eff2d5755a1a#diff-1c89fe2d0239918fff96d25555b8531fR359 even though it takes `ChainParams`. Why?

Reply:
There was a mistake while fixing commits. I'll fix it.